RELEASE.md

```diff
@@ -75,24 +75,29 @@ you can skip the dependency update or only update select dependencies. In such a
 case, you have to create an issue or pull request in the GitHub project for
 later follow-up.
 
-### Prepare your release
+### 1. Prepare your release
 
-For a patch release, work in the branch of the minor release you want to patch.
+Create a new release branch.
+
+For a patch release, work in the branch of the minor release you want to patch. So for 2.16.1, create a new branch off of 2.16.0.
 
 For a new major or minor release, create the corresponding release branch based on the master branch.
+** Note: ** this means that for a release candidate you should create the non-release candidate branch (release-2.16) and merge the RC versions (ex: cut-v2.16.0-rc.0) into it as you go.
```

Comment:
This explanation appears to be needlessly complicated to me.

I suggest to start with explaining the naming scheme for release branches: `release-x.y`. Then it's very easy to explain that all the RCs and patch releases belonging to a minor release go to the same branch.

Reply:
I'll reword this.